helmfile/charts/networkpolicy-service/templates/certmanager-np.yaml

```diff
@@ -0,0 +1,129 @@
+{{ if .Values.certManager.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: cert-manager-deny-all
+  namespace: cert-manager
+spec:
+  podSelector: {}
+  policyTypes:
+    - Ingress
+    - Egress
+
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: cert-manager-np
+  namespace: cert-manager
+spec:
+  podSelector: 
+    matchLabels:
+      app: cert-manager  
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:        
+    - from:
+        - namespaceSelector:
+            matchExpressions:
+              - key: kubernetes.io/metadata.name
+                operator: In
+                values:
+                  - monitoring
+        - podSelector:
+            matchLabels:
+              app.kubernetes.io/instance: kube-prometheus-stack-prometheus                    
+      ports:
+        - port: 9402 # promtheus metrics            
+  egress: 
+    - ports:
+      - port: 53
+        protocol: UDP        
+    - to:
+      {{- range $IP := .Values.certManager.letsencrypt.ips }}
+      - ipBlock:
+          cidr: {{ $IP }}
+      {{- end }}
+      ports:
+        - protocol: TCP
+          port: 443            
```

Comment:
Didn't we say to move all the stuff we don't control to configurable values? If so, please make this configurable